modules/effects/arrow-effects-kotlinx-coroutines/src/main/kotlin/arrow/effects/DeferredK.kt

```diff
@@ -533,6 +616,59 @@ sealed class DeferredK<A>(
         }
       }
   }
+
+  override val children: Sequence<Job>
```

Comment:
Isn't this covered by `Deferred<A> by memoized`?

Reply:
Not since we had to change `memoized` from `val` to `var`. Kotlin only has support for `val` but it also possible to use with `var`... It's a little bit nasty tho.

A short example of what kotlinc spits out:

```
public final class Sample implements Deferred {
   @NotNull
   private Deferred memoized;
   // $FF: synthetic field
   private final Deferred $$delegate_0;

   @NotNull
   public final Deferred getMemoized() {
      return this.memoized;
   }

   public final void setMemoized(@NotNull Deferred var1) {
      Intrinsics.checkParameterIsNotNull(var1, "<set-?>");
      this.memoized = var1;
   }

   public Sample(@NotNull Deferred memoized) {
      Intrinsics.checkParameterIsNotNull(memoized, "memoized");
      super();
      this.$$delegate_0 = memoized;
      this.memoized = memoized;
   }

   @NotNull
   public Sequence getChildren() {
      return this.$$delegate_0.getChildren();
   }

   public boolean isActive() {
      return this.$$delegate_0.isActive();
   }

   public boolean isCancelled() {
      return this.$$delegate_0.isCancelled();
   }
```